### PR TITLE
Group newsletters based on Group property

### PR DESCRIPTION
**Pull request by phillipbarron**
Newsletters in the MMA page are currently grouped using the theme property. This is different to the grouping logic in the [all Newsletters Page](https://www.theguardian.com/email-newsletters). 

## What does this change?

Updates the Newsletters type to include the group property which is [now returned from the identity API](https://github.com/guardian/identity/pull/2420) & uses this to group Newsletters instead of theme. The Colour mapping has been updated too. With the `News in Brief` & `News in Depth` sections using the News colour. 

## How to test

Verify changes as expected and Subscribe / unsubscribe behaviour as before

## Have we considered potential risks?

Behaviour very similar to previous state. Not expecting any issues. 
Needs a test in code but should be ok

## Images

**Before**

Screenshot 2023-07-04 at 08 37 50

**After**

Screenshot 2023-07-04 at 08 37 44

Screenshot 2023-07-04 at 08 38 41

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/mma/identity/idapi/newsletters.ts [hunk 1]
```diff
@@ -37,9 +40,11 @@ const newsletterToConsentOption = (
 
 export const read = async (): Promise<ConsentOption[]> => {
 	const url = '/newsletters';
-	return (await identityFetch<NewsletterAPIResponse[]>(url)).map(
-		newsletterToConsentOption,
+	const newslettersResponse = await identityFetch<NewsletterAPIResponse[]>(
+		url,
 	);
+	console.log({ idapiNewslettersResponse: newslettersResponse });
```
Comment (andrewHEguardian) [inline client/components/mma/identity/idapi/newsletters.ts:46]: should we be logging this?

**Comment by andrewHEguardian** [comment]
image
the storybook stories no longer show any newsletters because the mocked responses don't contain the `group` property

File: client/components/mma/identity/idapi/newsletters.ts (hunk 1 above)
Comment (phillipbarron) [reply client/components/mma/identity/idapi/newsletters.ts:46]: nope - debug - thank you

**Comment by phillipbarron** [comment]
> image the storybook stories no longer show any newsletters because the mocked responses don't contain the `group` property

Ahh cheers - I will update the fixture and re-check

**Review by andrewHEguardian — APPROVED** [review]
Looks good! I accepted the chromatic changes since it looked right

**Pull request merged**